docs/en_US/TrainingService/LocalMode.rst

```diff
@@ -17,8 +17,9 @@ You have an implementation for MNIST classifer using convolutional layers, the P
 
 To enable NNI API, make the following changes:
 
-* Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
-* Get predefined parameters
+   1.1 Declare NNI API: include ``import nni`` in your trial code to use NNI APIs.
```

Comment:
Might as well remove the indentation. As indentation is rst means "quote" I think.

Reply:
fix it